docs/content/api/index.ngdoc

```diff
@@ -8,6 +8,8 @@ Welcome to the AngularJS API docs page. These pages contain the AngularJS refere
 The documentation is organized into **{@link guide/module modules}** which contain various components of an AngularJS application.
 These components are {@link guide/directive directives}, {@link guide/services services}, {@link guide/filter filters}, {@link guide/providers providers}, {@link guide/templates templates}, global APIs, and testing mocks.
 
+There is also a {@link guide/index} with articles on various topics, and a list of external resources.
```

Comment:
Actually it's not only articles but also videos, tutorials (could be presentations). Maybe "resources" is more descriptive.